data/installed_data_test.go

```diff
@@ -0,0 +1,55 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/deis/workflow-manager/k8s"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/testapi"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/client/unversioned/testclient/simple"
+)
+
+const namespace = "deis"
+
+func TestInstalledDeisData(t *testing.T) {
+	client := getK8sClient(t)
+	k := k8s.NewResourceInterfaceNamespaced(client, namespace)
+	installedData := installedDeisData{
+		k8sResources: k,
+	}
+	_, _ = installedData.Get()
+	//TODO: we need to create a fake client interface that is a union of api+extension fake clients
```

Comment:
does this need doing before this patch is merged? if not, can you ensure there's an issue in for this TODO